src/memusage.h

```diff
@@ -38,8 +38,9 @@ template<typename X> static inline size_t DynamicUsage(X * const &v) { return 0;
 template<typename X> static inline size_t DynamicUsage(const X * const &v) { return 0; }
 
 /** Compute the memory used for dynamically allocated but owned data structures.
- *  For generic data types, this is *not* recursive. DynamicUsage(vector<vector<int> >)
- *  will compute the memory used for the vector<int>'s, but not for the ints inside.
```

Comment:
I think this comment is misleading at best. It *does* account for the array of ints ("the ints").

Reply:
oh, I just realized the example had nested vectors. allright, I'll revert this